### PR TITLE
feat: add ams rdvp

### PR DESCRIPTION
**Pull request by gfanton**
- Add announce argument to rdvp server
- Add new rdvp endpoint:
```
$ madns /dnsaddr/rdvp.berty.io/ipfs/12D3KooWHhDBv6DJJ4XDWjzEXq6sVNEs6VuxsV1WyBBEhPENHzcZ
/ip4/51.15.25.224/udp/4040/quic/p2p/12D3KooWHhDBv6DJJ4XDWjzEXq6sVNEs6VuxsV1WyBBEhPENHzcZ
/ip4/51.15.25.224/tcp/4040/p2p/12D3KooWHhDBv6DJJ4XDWjzEXq6sVNEs6VuxsV1WyBBEhPENHzcZ
```

### CODE REVIEW

**Review by Jorropo — APPROVED** [review]
LGTM

**Review by Jorropo — APPROVED** [review]
LGTM

File: go/internal/config/berty.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ var BertyDev = &BertyConfig{
 	Bootstrap: config.DefaultBootstrapAddresses,
 	RendezVousPeers: []string{
 		"/dnsaddr/rdvp.berty.io/ipfs/QmdT7AmhhnbuwvCpa5PH1ySK9HJVB82jr3fo1bxMxBPW6p",
+		"/dnsaddr/rdvp.berty.io/ipfs/12D3KooWHhDBv6DJJ4XDWjzEXq6sVNEs6VuxsV1WyBBEhPENHzcZ",
```
Comment (Jorropo) [inline go/internal/config/berty.go:22]: Rebase and use IPs, not dns addr (you can copy the mobile config).

**Review by Jorropo — CHANGES_REQUESTED** [review]
not dns addr

**Review by Jorropo — APPROVED** [review]
LGTM (tested, works fine)

**Review by moul — APPROVED** [review]

**Pull request merged**